aces/Loris
# [SQL] Setting default value of ScannerID in mri_protocol to be NULL instead of 0

This PR sets the default value of ScannerID in the mri_protocol to be NULL. Before these changes, the default value was 0. This PR addresses Issue #7476 and should not be merged unless [#640](https://github.com/aces/Loris-MRI/pull/640) is ready to be merged as well.

**Testing instructions**
1. Run the following query to check which entries have ScannerID=0.
```SQL
SELECT * FROM mri_protocol WHERE ScannerID=0;
```
2. Source the file `2021-06-23_set_default_ScannerID_to_NULL.sql` 
```SQL
source SQL/New_patches/2021-06-23_set_default_ScannerID_to_NULL.sql
```
3. Check that ScannerID is NULL for the entries that had ScannerID=0 from step 1.
4. Rerun RaisinBread installation process and check that it works.
